tests/test_e2e_10_mef_eline.py

```diff
@@ -13,6 +13,11 @@
 
 TIME_FMT = "%Y-%m-%dT%H:%M:%S+0000"
 
+# BasicFlows
+# Each should have at least 3 flows, considering topology 'ring':
+# - 01 for LLDP
+# - 02 for amlight/coloring (node degree - number of neighbors)
+BASIC_FLOWS = 3
```

Comment:
@italovalcy, future suggestion to minimize the coupling with introducing a new NApp that might have its flows without requiring to refactor other tests:

- Instead of checking he entire `dump-flows` output, filter for only things that the particular test is interested in verifying. For instance, if mef_eline is only concerned with its flows [its prefix cookie](https://github.com/kytos-ng/mef_eline/blob/master/settings.py#L24):

`ovs-ofctl -O OpenFlow13 dump-flows s1 cookie=0xaa00000000000001/0xff00000000000000`

Let me know what you think.

Reply:
sounds good to me! thanks for the suggestion, @viniarck